03_Modern/11_initializers.rst

```diff
@@ -27,19 +27,18 @@ conditional operations based on this value. Let's take a look at the example pse
       // do something else
    }
 
-There is nothing wrong with this code it can be improved in some specific situation. 
+There is nothing wrong with this code, but it can be improved in specific situations. 
 
-You can see, that in shown example, the variable :code:`var` is used only for this if-else 
-statement. At the same time it leaks into the surrounding scope. In that case we need to deal with 
-it, and make sure to not mix it up with other variables.
+As shown in the example, the variable :code:`var` is used only for this if-else 
+statement. At the same time, it leaks into the surrounding scope. In that case, we need to make sure to not mix it up with other variables.
 
-Also, if the compailer can explicitly know that this variable will be used only there, it may 
-optimize the code better.
+Also, if the compiler can explicitly know that this variable will only be used there, it may 
+optimize the code better.[[[Does this mean it will optimize the code better than it did before? seems redundant, suggest deleting "better"]]]

```

Comment:
Also, if the compiler can explicitly know this variable will only be used in this context it can potentially produce more optimized code.